### PR TITLE
Debug t

### PR DESCRIPTION
**Pull request by Anthony69600**
Hello Thomas,

Corrections:
- business form
- spelling correction
- remove 2 buttons on the return on the audit part 2/3 and 3/3
- ActionFixtures : remove the lorem ipsum

Thank you

### CODE REVIEW

File: migrations/Version20220204135456.php [hunk 1]
```diff
@@ -0,0 +1,91 @@
+<?php
```
Comment (aldaitzwild) [inline migrations/Version20220204135456.php]: No, You don't update any entity, you shouldn't send me a new Migration. Don't send this file

**Review by aldaitzwild — CHANGES_REQUESTED** [review]

File: migrations/Version20220204135456.php (hunk 1 above)
Comment (Anthony69600) [reply migrations/Version20220204135456.php]: I removed the migration, thank you

File: src/Form/ProfilCommercialType.php [hunk 2]
```diff
@@ -81,7 +81,7 @@ public function buildFormLastPart(FormBuilderInterface $builder): void
     {
         $builder
         ->add('numberClosPerMonth', ChoiceType::class, [
-            'label' => 'Combien de closing fetes-vous par mois ?',
+            'label' => 'Combien de contrats fetes-vous par mois ?',
```
Comment (aldaitzwild) [inline src/Form/ProfilCommercialType.php]: Maybe without french errors, it would be better ...

**Review by aldaitzwild — CHANGES_REQUESTED** [review]

File: src/Form/ProfilCommercialType.php (hunk 2 above)
Comment (Anthony69600) [reply src/Form/ProfilCommercialType.php]: Sorry

**Review by aldaitzwild — APPROVED** [review]

**Pull request merged**